### ISSUE TITLE
AUDIO5 is not detected

### ISSUE THREAD

**Issue by MichaelHeimann** [mvdwetering/yamaha_ynca#277]
**Describe the bug**
Homeassistant shows Log Warning (see Warning Section) under normal operation. I do not know if and how I triggered the warning or if it's just because the integration is used.

Also: The mediaplayer doesn't offer AUDIO5 as a input

**To Reproduce**
Have a RX-V685

**Expected behavior**
No Warning

**Diagnostics**
[config_entry-yamaha_ynca-c75a667929ce94b932caa36327042a0f.json](https://github.com/mvdwetering/yamaha_ynca/files/14977744/config_entry-yamaha_ynca-c75a667929ce94b932caa36327042a0f.json)

**Logging**
```
Logger: ynca.enums
Source: /usr/local/lib/python3.12/site-packages/ynca/enums.py:215
First occurred: 9:57:05 AM (1 occurrences)
Last logged: 9:57:05 AM

Unknown value 'AUDIO5' in Input
```
**Environment**
Provide details about the versions you are using, which helps to reproduce and find the issue quicker.

- Home Assistant Core version: 2024.4.3
- Home Assistant operating environment (OS/Container/Supervised/Core): OVA VM
- Integration version: v7.7.1
- Last working version (if known): never?

**Additional context**
I'm not 100% sure, but I think AUDIO5 is the virtual input channel used for ARC. So maybe all receivers with ARC functionallity are affected.

**Comment by mvdwetering** [comment]
According to the manual AUDIO5 is a stereo analog audio input on the back of the receiver.

The current list of audio inputs only goes up to AUDIO4 so need to extend that (which is pretty easy), but next to that I have to figure out a way to avoid AUDIO5 getting enabled for all users of the integration. 

I need to think a bit about hos to do the second part.

**Comment by mvdwetering** [comment]
That was actually easier than expected.
AUDIO5 input should be available in v7.8.0

**Comment by mvdwetering** [comment]
Closing as it has been implemented
